utils/controleval.py

```diff
@@ -3,6 +3,8 @@
 import collections
 import json
 import os
+import yaml
+from prometheus_client import CollectorRegistry, Gauge, write_to_textfile
```

Comment:
We will need to add this requirements.txt

Reply:
Added a commit updating the requirements.txt